tests/daemon/test-daemon.c

```diff
@@ -185,9 +179,6 @@ process_logind_line (GString *line,
 {
```

Comment:
This comment got orphaned since it wasn't on a diff line, but I removed the logind methods